examples/eestream/store/main.go

```diff
@@ -68,13 +68,20 @@ func Main() error {
 	errs := make(chan error, len(readers))
 	for i := range readers {
 		go func(i int) {
-			fh, err := os.Create(
-				filepath.Join(flag.Arg(0), fmt.Sprintf("%d.piece", i)))
+			piecefile := filepath.Join(flag.Arg(0), fmt.Sprintf("%d.piece", i))
+			fh, err := os.Create(piecefile)
 			if err != nil {
 				errs <- err
 				return
 			}
-			defer fh.Close()
+
+			defer func() {
+				err := fh.Close()
+				if err != nil {
+					fmt.Println(err)
+				}
+			}()
```

Comment:
Can you do something about these deferred closes, so they don't look that ugly?

For the production code we have `utils.LogClose`, but it uses the zap logger to log any errors. Can we do anything similar for the examples too?

Reply:
I could make a func for each of the examples. I tried to make as little modifications as possible.